src/scaffold/zome/coordinator.rs

```diff
@@ -72,9 +72,9 @@ pub fn initial_lib_rs(dependencies: Option<&Vec<String>>) -> TokenStream {
         pub enum Signal {}
 
         /// Whenever an action is committed, we emit a signal to the UI elements to reactively update them
+        /// Don't modify the for loop if you want the scaffolding tool to generate appropriate signals for your entries and links
         #[hdk_extern(infallible)]
         pub fn post_commit(committed_actions: Vec<SignedActionHashed>) {
-            /// Don't modify this loop if you want the scaffolding tool to generate appropriate signals for your entries and links
```

Comment:
Was this comment moved due to clippy? It seems more likely to me that people would actually read this if the comment is right above the for loop. What about converting it into a double-shlash "normal" `//` comment instead of a doc comment? Or does it get removed then when parsing?

Reply:
It was getting removed, but I figured out the underlying root cause, fixed now